### PR TITLE
issue/1776: grunt translate tasks detached from src/course

### PR DESCRIPTION
**Pull request by oliverfoster**
#1776 

* allows the translate task to work on json files outside of the src/course style (i.e. AT style)
* allows the ``languagefiles`` destination to be changed using ```grunt --languagedir=/newplace/```

### CODE REVIEW

**Comment by moloko** [comment]
OK so I can export to a different folder no problem using `$ grunt translate:export --languagedir=export_for_localisation` - but I can't figure out the right command for importing back in again...

Assuming I have my newly-localised-to-german CSV files in export_for_localisation/de, and I run `grunt translate:import --targetLang=de --languagedir=export_for_localisation`, I get the following:
```bash
Running "_getTranslateConfig" task

Running "_loadLanguageFiles" task

Running "_loadMasterCourse" task
Warning: de folder already exist, to replace the content in this folder, please add a --replace flag to the grunt task. Use --force to continue.

Aborted due to warnings.
```
What am I missing?

**Comment by oliverfoster** [comment]
I think you might need to get rid of ``course/de`` or use ``--replace`` to overwrite it.

**Comment by moloko** [comment]
Gah just realised that the course I was using to test this was the same one I had set up for working on https://github.com/adaptlearning/adapt-contrib-spoor/pull/163 - so had a 'de' folder in src/course already!

sorreee

File: grunt/tasks/translate/exportFile.js [hunk 1]
```diff
@@ -9,7 +9,10 @@ module.exports = function (grunt) {
     
     var next = this.async();
     
-    grunt.file.mkdir("languagefiles/"+grunt.config("translate.masterLang"));
+    var outputFolder = path.join(process.cwd(), "languagefiles", grunt.config("translate.masterLang"));
```
Comment (tomgreenfield) [inline grunt/tasks/translate/exportFile.js]: For example,
```js
var languageDir = grunt.option("languagedir");

var outputFolder = languageDir ?
    path.join(languageDir, grunt.config("translate.masterLang")) :
    path.join(process.cwd(), "languagefiles", grunt.config("translate.masterLang"));
```

**Comment by moloko** [comment]
@oliverfoster you have conflicts I'm afraid

**Pull request merged**